docs/src/user_docs/stdlib/main.md

```diff
@@ -10,19 +10,6 @@ The second goal can be achieved because calls to procedures in the standard libr
 ## Organization and usage
 Procedures in Miden standard library are organized into modules, each targeting a narrow set of functionality. Modules are grouped into higher-level namespaces. However, higher-level namespaces do not expose any procedures themselves. For example, `std::math::u64` is a module containing procedures for working with 64-bit unsigned integers. This module is a part of the `std::math` namespace. However, the `std::math` namespace does not expose any procedures.
 
-To invoke a procedure from a standard library module, the module first needs to be imported using a `use` statement. Once a module is imported, procedures from this module can be invoked via the regular `exec` instruction as `exec.<module>::<label>` where `label` is the name of the procedure. An example of this is shown below.
-
-```
-use std::math::u64
-
-begin
-    push.1.0
-    push.2.0
-    exec.u64::checked_add
-end
-```
-In the above example we first push two 64-bit integers on the the stack, and then invoke a 64-bit addition procedure from `std::math::u64` module.
-
```

Comment:
If we're removing this, it might be helpful to link to the new section describing how to provide the stdlib to the assembler.

Reply:
Added a link!